Neos.Flow/Configuration/Settings.DocTools.yaml

```diff
@@ -20,34 +18,11 @@ Neos:
         packageKeys:
           - 'Neos.Flow'
           - 'Neos.Party'
-          - 'Neos.FluidAdaptor'
           - 'Neos.Kickstart'
           - 'Neos.Welcome'
         savePathAndFilename: '%FLOW_PATH_PACKAGES%Framework/Neos.Flow/Documentation/TheDefinitiveGuide/PartV/CommandReference.rst'
 
     references:
-      'TYPO3Fluid:ViewHelpers':
```

Comment:
Not entirely sure about that part. We should maybe keep providing the ViewHelper reference

Reply:
Move it to the Settings.yaml of the fluid adaptor package?

Reply:
Added it here
https://github.com/neos/flow-development-collection/pull/2152/files#diff-0b2ac3d01faed426f2d878ae4c6eebc7R22

and checked the output of `./flow configuration:show --type Settings --path Neos.DocTools` 👍